builders/interoperability/xcm/fees.md

```diff
@@ -292,10 +292,10 @@ ExecutionTime = (Weight / Picosecond) * NumberOfInstructions
 
 |                                                                                                          指令                                                                                                           |                             重量                              |
 |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------:|
-| [`ReserveAssetDeposited`](https://github.com/moonbeam-foundation/moonbeam/blob/{{ networks.moonbeam.spec_version }}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L70){target=_blank} |              {{ xcm.fungible_weights.display }}               |
-|      [`ClearOrigin`](https://github.com/moonbeam-foundation/moonbeam/blob/{{ networks.moonbeam.spec_version }}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_generic.rs#L547){target=_blank}      |        {{ xcm.generic_weights.clear_origin.display }}         |
-|     [`BuyExecution`](https://github.com/moonbeam-foundation/moonbeam/blob/{{ networks.moonbeam.spec_version }}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_generic.rs#L484){target=_blank}      |    {{ xcm.generic_weights.buy_exec.total_weight.display }}    |
-|     [`DepositAsset`](https://github.com/moonbeam-foundation/moonbeam/blob/{{ networks.moonbeam.spec_version }}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L59){target=_blank}      |              {{ xcm.fungible_weights.display }}               |
+| [`ReserveAssetDeposited`](https://github.com/moonbeam-foundation/moonbeam/blob/{{ networks.moonbeam.spec_version }}/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L71){target=_blank} |              {{ xcm.fungible_weights.display }}               |
```

Comment:
Can you double check these links? The below 4 are 404ing for me when building the site with `./buildSite.sh -c eshaben/xcm-overhaul -e eshaben/xcm-overhaul`

Reply:
good catch! thanks!